internal/accounts/create.go

```diff
@@ -213,32 +228,12 @@ func createInteractive(state *flowkit.State, loader flowkit.ReaderWriter) (*flow
 
 		address = account.Address
 	} else {
-		var link string
-		switch selectedNetwork {
-		case config.DefaultTestnetNetwork():
```

Comment:
This shouldn't be removed. User should be able to choose a provider for account creation once they choose the network. In case of mainnet it should be blockport or lillico.

Reply:
addressed

Reply:
@sideninja Since there is issue with getting transactions from testnet, let's just keep the current implementation of monitoring the events and get the account that matches the public key to get the new account. What do you think?

Reply:
I wouldn't do that. If there's an issue on testnet that will be fixed I'm sure, we shouldn't work around that. What is the issue by the way?